src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -40,79 +39,75 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
         /// <summary>
         /// Maps format document code cleanup options to DiagnosticId[]
         /// </summary>
-        private static ImmutableArray<(string description, PerLanguageOption<bool> option, ImmutableArray<string> diagnosticIds)> _optionDiagnosticsMappings =
+        private static ImmutableArray<(DiagnosticSet diagnosticSet, PerLanguageOption<bool> option)> _optionDiagnosticsMappings =
```

Comment:
not going to make you change this, but hte ordering here seems reversed from what i would expect.  in other words, i'd expect hte option, and the set of diagnostics that correspond to it.

Reply:
I'm planning to clean this up in #30605, including but not limited to removing the option definitions.

Reply:
great.  thanks!